worlds/AutoWorld.py

```diff
@@ -334,7 +334,7 @@ def __getattr__(self, item: str) -> Any:
 
     # overridable methods that get called by Main.py, sorted by execution order
     # can also be implemented as a classmethod and called "stage_<original_name>",
-    # in that case the MultiWorld object is passed as an argument, and it gets called once for the entire multiworld.
+    # in that case the MultiWorld object is passed as first argument, and it gets called once for the entire multiworld.
```

Comment:
``suggestion
    # in that case the MultiWorld object is passed as the first argument, and it gets called once for the entire multiworld.
```